packages/components/tokens/components/button.js

```diff
@@ -45,8 +45,8 @@ module.exports = {
   },
 
   primary: {
-    background: { value: `${elements.color.primary}` },
-    color: { value: `${elements.color.white}` },
+    background: { value: `${elements.color.primary.base}` },
+    color: { value: `${elements.color.white.base}` },
```

Comment:
There should be no `.base` color, we decided to use `.main`

Reply:
Fixed!